### PR TITLE
PD-1446 / 25.04 / PD-1446 Update Active Directory Content

### PR DESCRIPTION
**Pull request by micjohnson777**
This PR updates the UI ref and tutorial articles with new screenshots and content on Active Directory screen settings and configuration instructions.

Backport this PR to 24.10.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

**Comment by micjohnson777** [comment]
Requesting a review from Andrew Walker to make sure the changes are accurate technically.

File: content/SCALE/SCALETutorials/Credentials/DirectoryServices/ConfigADSCALE.md [hunk 1]
```diff
@@ -23,34 +25,46 @@ Joining an AD domain also configures the Privileged Access Manager (PAM) to let
 
 Users can configure AD services on Windows or Unix-like operating systems using [Samba version 4](https://wiki.samba.org/index.php/Setting_up_Samba_as_an_Active_Directory_Domain_Controller#Provisioning_a_Samba_Active_Directory).
 
-To configure an AD connection, you must know the AD controller domain and the AD system account credentials.
+### Preparing to Configure AD in TrueNAS
+Before configuring Active Directory (AD) in TrueNAs:
 
-### Preparing to Configure AD in SCALE
+you must know the AD controller domain  name and the AD administration account credentials.
+You also need to know the hostname assigned to the TrueNAS system.
```
Comment (anodos325) [inline content/SCALE/SCALETutorials/Credentials/DirectoryServices/ConfigADSCALE.md:32]: No, you don't. We generate a computer account on domain join.

File: content/SCALE/SCALETutorials/Credentials/DirectoryServices/ConfigADSCALE.md [hunk 2]
```diff
@@ -23,34 +25,46 @@ Joining an AD domain also configures the Privileged Access Manager (PAM) to let
...
 
-Users can take a few steps before configuring Active Directory (AD) to ensure the connection process goes smoothly.
+* Obtain the AD admin account name and password.  
 
-1. Obtain the AD admin account name and password.  
+* [Verify name resolution](#verifying-name-resolution).
+  Go to **Network > Global Network Settings** to verify your TrueNAS network DNS name servers are configured to access the target domain controller you plan to add on the **Active Directory** screen.
```
Comment (anodos325) [inline content/SCALE/SCALETutorials/Credentials/DirectoryServices/ConfigADSCALE.md:37]: No. That's wrong. The name servers must be domain controllers.

File: content/SCALE/SCALETutorials/Credentials/DirectoryServices/ConfigADSCALE.md [hunk 3]
```diff
@@ -23,34 +25,46 @@ Joining an AD domain also configures the Privileged Access Manager (PAM) to let
...
+
+Go to **Credentials > Users**, locate the administrator account, click **Edit** and scroll down to the **Shell** setting.
+If not set to a shell option, select **TrueNAS Console**, and click **Save**.
+This opens the TrueNAS **Shell** screen showing the Console Setup menu. 
+
+Return to the **Shell** screen and enter the option to open the Linux shell and press <kbd>Enter</kbd>.
+{{< /hint >}}
 
-To confirm that name resolution is functioning, you can use the **Shell** and issue a `ping` command and a command to check network SRV records and verify DNS resolution.
+To confirm that name resolution is functioning, open the TrueNAS **Shell** and issue a `ping` command and then a command to check network SRV records and verify DNS resolution.
```
Comment (anodos325) [inline content/SCALE/SCALETutorials/Credentials/DirectoryServices/ConfigADSCALE.md:57]: We perform DNS checks during domain join, but if we're going to provide examples they should include dig commands that explicitly look up relevant AD SRV records. Simple ping is misleading.

**Comment by micjohnson777** [comment]
Suggested changes made to the ConfigADSCALE.md and ActiveDirectory.md articles.

**Review by DjP-iX — APPROVED** [review]

**Pull request merged**